pymaid/core.py

```diff
@@ -1632,6 +1660,7 @@ def __getattr__(self, key):
                      'n_postsynapses', 'n_open_ends', 'n_end_nodes',
                      'cable_length', 'tags', 'igraph', 'soma', 'root',
                      'segments', 'graph', 'n_branch_nodes', 'dps',
+                     'connector_tags',
```

Comment:
I don't think we need to purge connector_tags here. That would mean that they are cleared whenever changes to the neuron's topology/morphology are made. I do realise `tags` is listed there too but that should probably not be in there either?

Reply:
I don't fully understand what this block does (it looks to me like CatmaidNeuron's getattr function forces a purge of tags already upon each access?) but happy to do what you think is best. Want me to remove `tags` and `connector_tags` from this list?

Reply:
There are a few neuron properties that are computed on-the-fly: for example the graph representation of the neuron. Once generated they are cached. In the current implementation we need to manually invalidate those cached properties whenever making changes to the underlying structure (i.e. the node table). The dev branch has an automatic cache invalidation using hash values that works quite nicely but for now: yes, removing `tags` and `connector_tags` from that block would be the right thing to do.